package.json

```diff
@@ -10,6 +10,7 @@
     "@types/node": "^16.11.45",
     "@types/react": "^18.0.15",
     "@types/react-dom": "^18.0.6",
+    "@types/styled-components": "^5.1.25",
```

Comment:
Typy dla styled-components były już zainstalowane w devDependencies

Reply:
Tak, zgadza się, to było w pull requestach z dzisiaj, więc ja tego nie miałem jeszcze u siebie.